xwiki/content.md

```diff
@@ -106,17 +106,188 @@ services:
       - MYSQL_DATABASE=xwiki
     networks:
       - bridge
+# How to use this image
```

Comment:
Uh, I think we've entered a time loop here -- this content looks very similar to the content above, but is now duplicated instead of replaced.  I don't think that was intentional. :smile: